modules/userId/index.js

```diff
@@ -820,6 +821,17 @@ export function init(config) {
       auctionDelay = utils.isNumber(userSync.auctionDelay) ? userSync.auctionDelay : NO_AUCTION_DELAY;
       updateSubmodules();
     }
+    // userSync.ppid should be one of the 'source' values in getUserIdsAsEids() eg pubcid.org or id5-sync.com
+    if (userSync && userSync.userIds && userSync.ppid && findIndex(getUserIdsAsEids(), x => x.source === userSync.ppid) >= 0 && typeof getUserIdsAsEids().find(x => x.source === userSync.ppid).uids[0].id === 'string') {
+      let googletag = window.googletag || {};
```

Comment:
Out of curiosity, why not

```js
window.googletag = window.googletag || { cmd: [] };
```

and then later reference via `window.googletag` ? :smiley:

Reply:
I was just following an example in another commit,  is there an advantage to your suggestion?

Reply:
Initialization is done in a single step, no intermediate variable, explicit referencing the googletag object (no shadowing).

I have been biten by JS so many times that I try to make things as explicit as possible :sweat_smile:

Reply:
just changed, can you confirm as per your suggestion?

Reply:
LGTM :heart: